_issues/emails.md

```diff
@@ -1,23 +1,23 @@
 ---  
 layout: post  
-title: "But isn't she a criminal for using a private email server?"  
+title: "Hillary is a criminal for using a private email server?"  
```

Comment:
Remove remove the "?" as well? Also, part of me thinks "She is" might be better than "Hillary is" here.